Firmware code/smart_energy_meter/src/token_management.hpp

```diff
@@ -6,7 +6,7 @@
 // Arduino base libraries
 
 // third party libraries
-#include <Keypad.h>
+#include "../.pio/libdeps/bluepill_f103c8_128k/Keypad/src/Keypad.h"
```

Comment:
This shouldn't be here 🤔 Looks like something is not working correctly on your setup, can you double check?

Reply:
OK thanks for your answer